app/views/WhatDoYouWantToDoTileView.scala.html

```diff
@@ -33,14 +35,9 @@
         ec: ExecutionContext)
 
 @optimizelyAudience = @{
-    if(model.showTaxCodeChangeTile()) {
-        Some("TaxCodeChangeUser")
-    } else {
-        None
-    }
+    model.maybeMostRecentTaxCodeChangeDate.map(_ => "TaxCodeChangeUser")
```

Comment:
I don't think this is used. Can someone correct me if wrong?

Reply:
@masahmad im not aware of any need to have optimizely stuff in our code are you?

Reply:
removed